src/opds_dumper.cpp

```diff
@@ -83,22 +83,22 @@ BookData getBookData(const Library* library, const std::vector<std::string>& boo
   return bookData;
 }
 
-struct LangInfo {
-  std::string selfName, englishName;
-};
-
-std::map<std::string, LangInfo> langMap = {
-      {"eng", { "English",  "English"} },
-      {"fra", { "Français", "French"} },
-      {"rus", { "Русский",  "Russian"} },
-};
-
 std::string getLanguageSelfName(const std::string& lang) {
-  return langMap.at(lang).selfName;
+  const icu::Locale locale(lang.c_str());
+  icu::UnicodeString ustring;
+  locale.getDisplayLanguage(locale, ustring);
+  std::string result;
+  ustring.toUTF8String(result);
+  return result;
 };
 
 std::string getLanguageEnglishName(const std::string& lang) {
-  return langMap.at(lang).englishName;
+  const icu::Locale locale(lang.c_str());
+  icu::UnicodeString ustring;
+  locale.getDisplayLanguage(icu::Locale("en"), ustring);
+  std::string result;
+  ustring.toUTF8String(result);
+  return result;
 };
```

Comment:
I wonder about the performance of this (although not really important).
Creating the icu::Locale may be time conssuming (to check).
Maybe we could have only one method returning a pair of string and so get the locale only once.

Reply:
What about simply dropping `getLanguageEnglishName()`. @kelson42 [considers the current format of the language entry pretty verbose](https://github.com/kiwix/libkiwix/pull/553#pullrequestreview-682387544), so removing the `<content>` node will solve both concerns at the same time.